src/components/TracePage/TraceTimelineViewer/ListView/__snapshots__/index.test.js.snap

```diff
@@ -42,7 +42,6 @@ ShallowWrapper {
                 style={
                     Object {
                         "height": 2,
-                        "overflow": "hidden",
```

Comment:
what is the purpose of these `__snapshots__` files? Are they auto-generated?

Reply:
Yep, https://facebook.github.io/jest/docs/en/snapshot-testing.html

Reply:
Oh man, my reply was posted to the wrong thread :/

The snapshots are for tests.